packages/react-examples/tsconfig.cy.json

```diff
@@ -3,6 +3,7 @@
   "compilerOptions": {
     "isolatedModules": false,
     "types": ["node", "cypress", "cypress-real-events"],
+    "typeRoots": ["../../../node_modules", "../../../node_modules/@types"],
```

Comment:
without explicit typeRoots to node_modules cannot find types for cypress and cypress-real-events